lib/utils.bash

```diff
@@ -622,9 +626,7 @@ with_plugin_env() {
   local path exec_paths
   exec_paths="$(list_plugin_exec_paths "$plugin_name" "$full_version")"
 
-  # exec_paths contains a trailing newline which is converted to a colon, so no
-  # colon is needed between the subshell and the PATH variable in this string
-  path="$(tr '\n' ':' <<<"$exec_paths")$PATH"
+  path="${exec_paths}:$PATH"
```

Comment:
I didn't find the comment to be true in testing, and this works fine as-is. Open to proof to the contrary.